specification/translation/data-plane/Azure.AI.DocumentTranslation/readme.md

```diff
@@ -0,0 +1,54 @@
+# Cognitive Services Translation Document SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating Translation Document SDK.
+
+The current release is `v1_0_Preview_1`.
+
+``` yaml
+tag: v1_0_Preview_1
+add-credentials: true
+openapi-type: data-plane
+```
+
+# Releases
+
+### Release 3.0
+These settings apply only when `--tag=v1_0_Preview_1` is specified on the command line.
+
+``` yaml $(tag) == 'v1_0_Preview_1'
+input-file: preview/2023-11-01-preview/openapi.json
+```
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+ ```
```

Comment:
You have an errant space here, which will break tools. Backticks in code fences need to be aligned.